on_demand/api/user-api.js

```diff
@@ -429,13 +429,57 @@ router.get('/draft/_id/:_id', (req, res, next) => {
       if (err) return next(err);
       cleanDraftRecord(result)
       console.log(result)
-      if (!req.authorizedTrackers.includes(result.trackerIDHash)) res.status(401).send({"error": "not authorized"})
+      if (!request.public && !req.authorizedTrackers.includes(result.trackerIDHash)) res.status(401).send({"error": "not authorized"})
```

Comment:
Hmmm I was hoping this would allow anonymous access, but it looks like something is noticing that there is no JWT in the call and I'm getting the 401 early anyway?

Would we want to make "shareable" entities separate anon-api routes?

Reply:
Oh, yeah, there's middleware here that requires that the JWT checks out: https://github.com/mtgatracker/mtgatracker-webtask/blob/master/on_demand/mtga-tracker-game.js#L170

(See also: lines 162-167)

I'm not sure what the best way to handle this is. I think you're right though and that the "I'm signed in" draft view (`/draft/?id=xyz`) should keep using this route, and there's also a new page for sharing (`/share/draft/?I'd=xyz`) that instead checks if the requested resource is published and doesn't have a jwt check.

Also, tbh I think we should drop the anon-api route altogether and just call them what they are, public routes. (obscurity != security)